pages/regions.js

```diff
@@ -0,0 +1,67 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { nanoid } from 'nanoid';
+
+import Layout from '../src/components/layout';
+import { Hero, Icon, P, H6 } from '../src/components/jambonz-ui';
+import { getData } from '../src/lib/data';
+
+function Regions({ data }) {
+  const router = useRouter();
+  const redirect = router.query.redirect;
+
+  return (
+    <section className="regions">
+      {data.regions.map((region) => {
+        const href = (redirect === 'login') ? region.url : region.altUrl;
+
+        return (
+          <div key={nanoid()} className="wrap regions__wrap">
+            <div className="regions__icon">
+              <Icon name={region.icon} subStyle={region.color} mainStyle="fill" />
+            </div>
+            <div className="regions__title">
+              <P className="h6">
```

Comment:
Okay, here is what I am proposing for how we handle this typography. First, go ahead and add a new selector to the typography SASS file that looks like this:

```sass
.med {
  @include font-medium();
}
```

This gives us a new generic className to utilize for these situations. Then based on my other comments here and the notion of utilizing this new classname I would expect to see the regions__title and regions__text markup end up looking like this:

```js
<div className="regions__title">
  <P className={`med color--${region.color}`}>
    {region.title}
  </P>
</div>
<div className="regions__text">
  <P className="med">
    <Link href={href}>
      <a className="i" title={`jambonz ${region.title}`}>
        <span className="regions__text">{region.text}</span>
        <Icon name="ExternalLink" />
      </a>
    </Link>
  </P>
</div>
```